.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ https://github.com/blog/1506-closing-issues-via-pull-requests for more
 information.
 
 ### checklist
-- [ ] Make sure you read and understood the [CONTRIBUTING.md](https://github.com/xh3b4sd/anna/.github/CONTRIBUTING.md).
+- [ ] Make sure you read and understood the [CONTRIBUTING.md](/.github/CONTRIBUTING.md).
```

Comment:
AFAIK this is how dynamic links work. Originally it was `/.github.com/CONTRIBUTING.md`. The directory is simply `.github`. When I visit `.github/PULL_REQUEST_TEMPLATE.md` the link works. I am pretty sure this will also be the case for every pull request.